tcipostman/__init__.py

```diff
@@ -0,0 +1,25 @@
+from flask_mail import Mail, Message
+from tcidatabase import models
+from tcidatabase import db
+
+
+def init_app(app):
+    Postman.app = app
+    Postman.mail = Mail(app)
+    db.signals.post_save.connect(Postman.send_client_token,
+                                 sender=models.SingleToken)
+
+
+class Postman:
+    app = None
+    mail = None
+
+    @classmethod
+    def send_client_token(klass, sender, document, created, **kwargs):
```

Comment:
Right.Here's an explanation of each variable:

- `klass`: convention name for representing the current class in a class method. However, after checking [`cls` is declared preferred conventional name](https://www.python.org/dev/peps/pep-0008/)
- `sender, document, created`: arguments [called by the `post_save.connect()`](http://nullege.com/codes/show/src@m@o@mongoengine-0.8.7@mongoengine@document.py/301/mongoengine.signals.post_save.send)
- `kwargs`: extra arguments. They are not required here however.